### PR TITLE
WIP IC post

### PR DESCRIPTION
**Pull request by MiyamotoAkira**

### CODE REVIEW

File: src/_posts/2019-05-10-initiative-circles.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+abstract: Second post about internal workings of Codurance, concentrating on Initiative Circles.
+image: 
+    src: /assets/custom/img/blog/meeting2.png
+    attribution:
+       text: Photo by Sandro Viljoen
+tags: 
+- codurance
+---
+
+So this is the second post about the internal workings of Codurance. While in the previous one I looked at how salary and reviews work, here I want to concentrate on something that allows for me to take an active part in shaping our organisation: : Initiative Circles (IC from now on).
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-05-10-initiative-circles.md]: "the previous one" - perhaps add a link to that post here?

Comment (Mattsi-Jansky) [inline src/_posts/2019-05-10-initiative-circles.md]: "allows for me"- "for" is unnecessary

File: src/_posts/2019-05-10-initiative-circles.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+I have always wanted to make my life and the life of my colleagues better. There is no company that will not benefit from improvements, some far more than others. At Codurance, with Initiative Circles, anyone has the opportunity to improve the lives of their co-workers, from small improvements to much more significant changes. At Codurance, through ICs, we have addressed disparate topics  such as: Paid Slack, BitWarden for password keeping, the Progress Structure that is the subject of my next post, salary alignment with the local market, , Peer Review Process, The Podcasts, …
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-05-10-initiative-circles.md]: Maybe include health insurance in the list of past IC benefits, as it's a good one that shows how much an IC can influence people.

File: src/_posts/2019-05-10-initiative-circles.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+The Initiative Circles was an idea introduced by Sandro and Mash, our co-founders, with the vision to empower everyone to evolve the organisation according to the business and organisational needs. Their aim is to create a truly learning organisation. 
+
+IC are relatively simple to set up: someone has an itch to scratch, and floats the idea. If two other people like the idea and want to collaborate then the IC can proceed.  “Anything” can be tackled. You seek advice from people who will be impacted, reach a resolution, communicate to people, and go ahead and implement it. Rather than seeking complete consensus we work on trust that if enough people agree with an idea then the rest will trust them to have their best at heart. If people don’t like the results of an IC then they can create another IC and change its outcome.
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-05-10-initiative-circles.md]: "their best at heart" -> "their best interests at heart"?

Comment (Mattsi-Jansky) [inline src/_posts/2019-05-10-initiative-circles.md]: "change its outcome." -> "change it's outcome"

File: src/_posts/2019-05-10-initiative-circles.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+And I think I mislead you talking about the previous paragraph being the last thing to talk about. Like with so many things, it requires that people participate. But, unlike in any other company I have ever worked, you can be an agent of change and improvement. Before I was reduced to complain about the situation, now I can be the change. If I don’t, well, can’t complain, can I?
```
Comment (Mattsi-Jansky) [inline src/_posts/2019-05-10-initiative-circles.md]: Perhaps "Before I was reduced to complain about the situation" -> "Before I could only complain about bad situations"

**Review by Mattsi-Jansky — COMMENTED** [review]
Gave it a quick read, added some suggestions

File: src/_posts/2019-05-10-initiative-circles.md (hunk 3 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-05-10-initiative-circles.md]: Possesive, doesn't need apostrophe

File: src/_posts/2019-05-10-initiative-circles.md (hunk 2 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-05-10-initiative-circles.md]: there is even space between commas for it :-)

File: src/_posts/2019-05-10-initiative-circles.md (hunk 3 above)
Comment (Mattsi-Jansky) [reply src/_posts/2019-05-10-initiative-circles.md]: Doesn't it? I believe possessive nouns need apostrophes. In this case "it" is the IC, and it possesses the outcome. I may be missing something, but I did a quick google and I think it holds up my understanding: https://grammar.yourdictionary.com/punctuation/apostrophe-rules.html

File: src/_posts/2019-05-10-initiative-circles.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+And that is basically it. Simple, isn’t it?
+
+There is a bit more, of course. Which I will be describing now.
+
+Communication (something that we need to revisit as  Codurance grows) is important. We use Slack heavily, so there is a channel (conveniently called `#initiative-circles`) where any new initiatives appears when we add it on our tracker (currently we are using [Taiga](https://taiga.io)). Also, the person that wants the IC to happen usually links in `#general`, as not everyone is on the IC channel. We usually also create a channel specific for an IC once it has started. One of the benefits of this additional channel is that people that don’t want to have an active participation still can follow how the IC evolves. Finally, once the IC has been concluded, we let the rest of the company know. At the moment the rules says that you need to do it during catch-ups or openspaces (how cool is to have your own day long unconference every two months?). But that doesn’t scale, so we are going to be looking at improving that … through an IC.
```
Comment (christopher-bimson) [inline src/_posts/2019-05-10-initiative-circles.md]: double space here?
> as[sp][sp]Codurance grows)

Comment (christopher-bimson) [inline src/_posts/2019-05-10-initiative-circles.md]: appear?

File: src/_posts/2019-05-10-initiative-circles.md (hunk 3 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-05-10-initiative-circles.md]: that doesn't talk about the pronouns her, his or ... its :-)
https://grammarist.com/spelling/its-its/

File: src/_posts/2019-05-10-initiative-circles.md (hunk 5 above)
Comment (christopher-bimson) [inline src/_posts/2019-05-10-initiative-circles.md]: You might want to hyperlink Slack to their homepage, for the SEO juice.

File: src/_posts/2019-05-10-initiative-circles.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+And the conclusion is the last bit that I want to talk about. Sometimes an IC fails to take off the ground, and never starts. That’s fine. It wasn’t an itch worth scratching. Sometimes an IC fizzles out and never reaches a conclusion. That’s fine. Maybe the priority of the IC wasn’t that big, or maybe the effort was too much for the expected outcome. And sometimes the IC reaches a conclusion with a specific outcome. Then it gets implemented. The people actively taking part in the IC need to be in agreement. That is one of the two conditions that applies to all IC conclusions. It could be possible that people from outside the IC don’t agree, but they have had the opportunity to taking part. Also it is the responsibility of the IC members to seek advice from people impacted but they don’t need to follow everyone’s advice. The other condition is that you have done your due diligence, especially if there is a cost to it. You could have a preconceived idea about the conclusion, but is important to see as many alternatives as possible, in case you find a better outcome.
```
Comment (christopher-bimson) [inline src/_posts/2019-05-10-initiative-circles.md]: >to taking part

to take part?

File: src/_posts/2019-05-10-initiative-circles.md (hunk 3 above)
Comment (Mattsi-Jansky) [reply src/_posts/2019-05-10-initiative-circles.md]: Ahh, I forgot that "its" has an exception doesn't it. My bad. My dissertation supervisor used to catch me out on this one all the time 😅

File: src/_posts/2019-05-10-initiative-circles.md (hunk 5 above)
Comment (MiyamotoAkira) [reply src/_posts/2019-05-10-initiative-circles.md]: I actually changed iniatitives into singular

File: src/_posts/2019-05-10-initiative-circles.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    src: /assets/custom/img/blog/meeting2.png
+    attribution:
+       text: Photo by Sandro Viljoen
+tags: 
+- codurance
+---
+
+So this is the second post about the internal workings of Codurance. While in the [previous one](https://codurance.com/2019/05/02/salaries-at-codurance/) I looked at how salary and reviews work, here I want to concentrate on something that allows me to take an active part in shaping our organisation: Initiative Circles (IC from now on).
+
+I have always wanted to make my life and the life of my colleagues better. There is no company that will not benefit from improvements, some far more than others. At Codurance, with Initiative Circles, anyone has the opportunity to improve the lives of their co-workers, from small improvements to much more significant changes. At Codurance, through ICs, we have addressed disparate topics  such as: Paid Slack, BitWarden for password keeping, the Progress Structure that is the subject of my next post, salary alignment with the local market, health insurance , Peer Review Process, The Podcasts, …
```
Comment (christopher-bimson) [inline src/_posts/2019-05-10-initiative-circles.md]: >health insurance ,

Space before the comma

Comment (christopher-bimson) [inline src/_posts/2019-05-10-initiative-circles.md]: >disparate topics  such as: 

I think there is another double space there.

Comment (MiyamotoAkira) [reply src/_posts/2019-05-10-initiative-circles.md]: There was :-)

**Pull request merged**